doc/nrf/releases_and_maturity/releases/release-notes-2.7.0.rst

```diff
@@ -950,7 +961,21 @@ Bluetooth libraries and services
 
 * :ref:`bt_fast_pair_readme` library:
 
+  * Added support for the FMDN extension (:kconfig:option:`CONFIG_BT_FAST_PAIR_FMDN`).
+  * Added the :kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_AK_BACKEND` Kconfig option to select the Account Key storage backend and introduced the new minimal backend for Account Key storage.
+    The new backend (:kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_AK_BACKEND_MINIMAL`) is dedicated for the locator tag use case and supports the Owner Account Key functionality required by the FMDN extension.
+    The old backend (:kconfig:option:`CONFIG_BT_FAST_PAIR_STORAGE_AK_BACKEND_STANDARD`) is used in the input device use case.
+  * Added the Beacon Clock storage unit used with the FMDN extension.
+  * Added the Ephemeral Identity Key (EIK) storage unit used with the FMDN extension.
+  * Added support for AES-ECB-256, SECP160R1 and SECP256R1 cryptographic operations to the Oberon crypto backend (:kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_OBERON`) that are used in the FMDN extension.
   * Added experimental support for a new cryptographical backend that relies on the PSA crypto APIs (:kconfig:option:`CONFIG_BT_FAST_PAIR_CRYPTO_PSA`).
+  * Added the :kconfig:option:`CONFIG_BT_FAST_PAIR_REQ_PAIRING` Kconfig option to provide the possibility to skip the Bluetooth pairing and bonding phase during the Fast Pair Procedure.
+    This option must be disabled in the Fast Pair locator tag use case due to its requirements.
+  * Added the :kconfig:option:`CONFIG_BT_FAST_PAIR_GATT_SERVICE_MODEL_ID` Kconfig option to provide the possibility to remove the Model ID characteristic from the Fast Pair service.
+    This option must be disabled in the Fast Pair locator tag use case due to its requirements.
+  * Added an UUID definition of the FMDN Beacon Actions characteristic to the Fast Pair UUID header (:file:`bluetooth/services/fast_pair/uuid.h`)
```

Comment:
We could group the added into one bullet point like below:

* Added:
   
  * Support for the FDMN...
  * ...